app/src/components/Generation/GenerationForm.tsx

```
@@ -15,7 +15,9 @@ import { Input } from '@/components/ui/input';
 import {
   Select,
   SelectContent,
+  SelectGroup,
   SelectItem,
+  SelectLabel,
```

Biome (2.4.11)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)